src/librustc_passes/loops.rs

```diff
@@ -140,11 +141,13 @@ impl<'a, 'hir> Visitor<'hir> for CheckLoopVisitor<'a, 'hir> {
                                 .span_label(e.span,
                                             "can only break with a value inside \
                                             `loop` or breakable block")
-                                .span_suggestion(e.span,
+                                .span_suggestion_with_applicability(e.span,
                                                  &format!("instead, use `break` on its own \
                                                            without a value inside this `{}` loop",
                                                           kind.name()),
-                                                 "break".to_string())
+                                                 "break".to_string(),
+                                                 Applicability::Unspecified,
+                                )
```

Comment:
Fix indentation.

Reply:
Making 19 spaces back instead of 4 because of `format!` message approaches 100-character tidy line.